content/docs/command-reference/move.md

```diff
@@ -69,6 +71,46 @@ outs:
     path: other.csv
 ```
 
+### Manually renaming stage outputs
+
+In this example, let's rename stage output `keras.h5` to `model.h5`:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  train:
+    cmd: python train.py
+    ...
+    outs:
+    - keras.h5
+```
+
+First we need to change the name of model file in code (`train.py`). Next we
+change the output name in the `train` stage of `dvc.yaml`. After this we change
+`/keras.h5` to `/model.h5` in the `.gitignore` file. Lastly, we rename the
+existing model file:
+
+```dvc
+$ mv keras.h5 model.h5
+```
+
+In the final step we run `dvc commit` with `-f` option to force save the
+changes:
+
+```dvc
+$ dvc commit -f
+```
+
+Alternatively, we can also use `dvc run` with `-f` to overwrite the stage
+changes in `dvc.yaml`:
+
+```dvc
+$ dvc run -f -n train \
+          -d data.csv -d train.py -o model.h5 \
+          -p train.epochs,train.lr \
+          python train.py
+```
```

Comment:
Getting too long and this doesn't seem "manual" anymore (meaning not using DVC commands I guess, because everything here is manual in a way... — consider renaming the section if you can think of a better word BTW)

Reply:
Without the "Alternatively...." part now it's comparatively shorter. Also if it's in `dvc-files-and-dir` doc then the above `$cat`  block might be needed for context.